.github/workflows/manual-release.yml

```
@@ -0,0 +1,117 @@
+        type: string
+      indexImage:
+        description: 'Kuadrant operator index image (example: quay.io/kuadrant/kuadrant-operator-catalog:v1.4.3)'
+        required: true
+        type: string
+      ocpVersion:
+        description: 'Openshift version to pin operator versions for. (example: v4.22)'
+        required: true
+        default: v4.22
+
+env:
+  HELM_VERSION: ${{ inputs.version }}-${{ inputs.ocpVersion }}
```

actionlint (1.7.12)

[error] property "version" is not defined in object type {indeximage: string; kuadrantversion: string; ocpversion: any}

(expression)